requirements.txt

```diff
@@ -1,35 +1,33 @@
-aiohttp==3.9.5
+aiohappyeyeballs==2.4.4
+aiohttp==3.10.11
```

Comment:
Why did you bump to 3.10.11 here, where the dependabot PR says 3.11.10? Manual typo? Did you type this manually? (and not use the dependabot PR as template?) :D

Reply:
Was due to the github workflow stating 3.8, while this repo is 3.10. Weird issue introduced in the first commit. Fixed now.